server/server/views/recover.py

```diff
@@ -8,28 +10,42 @@
 
 from ..models.user import User
 
+
 @view_config(route_name='recover_password', renderer='json')
+#def password_generator(size=8, chars=string.ascii_letters + string.digits):
+    #return ''.join(random.choice(chars) for i in range(size))
+
 def recover_send_mail(request):
+    global temporary_password
+    random_charset = "abcdefghijklmnopqrstuvwxyz01234567890ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    temporary_password = "".join(random.sample(random_charset,8 ))
     json = request.json_body
     user = request.dbsession.query(User).filter_by(email=json['email']).one_or_none()
     if user.email is not None:
         token = request.dbsession.query(User).filter_by(email=json['email']).one_or_none()
         mailer = request.mailer
-        message = Message(subject="hello world",
+        message = Message(subject="Recover password",
                           sender="eventmerv@gmail.com",
                           recipients=[json["email"]],
-                          body='http://localhost:6543/change_password/')
+                          body='temporary password:'+ temporary_password +' http://localhost:6543/change_password/')
         mailer.send(message)
         mailer.send_immediately(message, fail_silently=False)
 
     return "We send link for change password in your mail", json['email']
 
+@resource(path='change_password', renderer='json')
```

Comment:
You don't need to define resource for one route with one request method. Better to use service for this. Also if you use Cornice, why did you create view by `@view_config`?